go.mod

```diff
@@ -1,66 +1,109 @@
 module github.com/openshift/certman-operator
 
-go 1.16
+go 1.17
 
 require (
-	github.com/Azure/azure-sdk-for-go v43.2.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.10.2
-	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2
-	github.com/Azure/go-autorest/autorest/to v0.3.1-0.20191028180845-3492b2aff503
-	github.com/aws/aws-sdk-go v1.34.19
+	github.com/Azure/azure-sdk-for-go v66.0.0+incompatible
+	github.com/Azure/go-autorest/autorest v0.11.24
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
+	github.com/Azure/go-autorest/autorest/to v0.4.0
+	github.com/aws/aws-sdk-go v1.44.87
 	github.com/eggsampler/acme v1.0.0
-	github.com/go-logr/logr v0.2.1
-	github.com/go-openapi/spec v0.20.3
-	github.com/gogo/protobuf v1.3.2 // indirect
-	github.com/golang-jwt/jwt/v4 v4.4.1 // indirect
-	github.com/lib/pq v1.2.0
-	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
-	github.com/openshift/aws-account-operator/pkg/apis v0.0.0-20210521135120-4790241bcdd5
-	github.com/openshift/hive v1.0.16-0.20201211144432-f97557354336
-	github.com/openshift/operator-custom-metrics v0.3.1-0.20200901174648-463079905232
-	github.com/operator-framework/operator-sdk v0.17.2
+	github.com/go-logr/logr v1.2.2
+	github.com/lib/pq v1.10.6
+	github.com/openshift/api v0.0.0-20220531073726-6c4f186339a7
+	github.com/openshift/hive/apis v0.0.0-20220831193253-7dca200fad2f
+	github.com/openshift/operator-custom-metrics v0.5.0
+	github.com/operator-framework/operator-lib v0.11.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.12.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.10.0
-	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.6.1
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	google.golang.org/api v0.25.0
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
-	sigs.k8s.io/controller-runtime v0.6.2
+	github.com/ravitri/aws-account-operator/api v0.0.0-20220902055620-a0d038965be6
```

Comment:
**NOTE**: This is temporary till https://github.com/openshift/aws-account-operator/pull/725 is merged. Once it is merged, migrated certman-operator to be tested again with any rebase changes of migrated AAO. If all works, will point to upstream AAO repository then.

Reply:
i don't have any significant concerns once this is resolved

thanks for the big PR!

Reply:
This has been updated now as the AAO PR is merged.